python/cuml/cuml/experimental/accel/__init__.py

```diff
@@ -32,12 +33,36 @@ def _install_for_library(library_name):
     importlib.import_module(f"._wrappers.{library_name}", __name__)
 
 
+def _is_concurrent_managed_access_supported():
```

Comment:
Should we note this was borrowed from cudf? Do we need to keep our implementation in line with theirs at all? Would it make sense to upstream this to cuda-python (or rmm?) at some point?

Reply:
I already tweaked it just _slightly_ from that implementation, and I don't think they necessarily need to be kept in sync. It's conceivable to me that we would want to check slightly different things for different libraries in the future. Since we do need exactly the same check right now, I wouldn't be opposed to upstreaming it somewhere though.